components/automate-ui/src/app/components/page-picker/page-picker.component.ts

```diff
@@ -11,15 +13,42 @@ export class PagePickerComponent implements OnChanges {
   @Input() perPage: number;
   @Input() page: number;
   @Input() maxPageItems = 10;
+  @Input() @HostBinding('class.forDesktop') forDesktop = false;
+  @Input() @HostBinding('class.fullScreened') fullScreened = false;
 
   @Output() pageChanged = new EventEmitter<number>();
+  @Output() pageSizeChanged = new EventEmitter<PageSizeChangeEvent>();
+
+  itemStartCount: number;
+  itemEndCount: number;
+  totalPages: number[];
```

Comment:
I ended up renaming the totalPages variable to allPages since its actually the array used to display the count, and will update the total number as suggested